python-sdk/docs/CHANGELOG.md

```diff
@@ -1,5 +1,24 @@
 # Changelog
 
+## 1.3.1
+
+### Feature:
+- Introduce a parameter `if_exists` which user can set to `append` or `replace` for `aql.dataframe()` operator. `append` insert into an existing table (`output_table`) from a Dataframe. [#1379](https://github.com/astronomer/astro-sdk/issues/1379)
+- Temporary Schemas accounts for Location/Region when creating BigQuery schema. User can now create schema in Bigquery as per region like - `<dataset_id>__<region_id>`. [#1369](https://github.com/astronomer/astro-sdk/issues/1369)
+
+### Bug fixes
+- Fix the `aql.cleanup()` operator as failing as the attribute `output` was implemented in 2.4.0 [#1359](https://github.com/astronomer/astro-sdk/issues/1359)
```

Comment:
This comment does not give much info from a User's perspective as they don't know what `output` attribute is?

It should be the following instead:

```suggestion
- Make `aql.cleanup()` operator compatible with Task Mapping features introduced in Airflow 2.4.0 [#1359](https://github.com/astronomer/astro-sdk/issues/1359)
```